mcodingbot/database/database.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import logging
+
+import apgorm
+
+from mcodingbot.config import CONFIG
+from mcodingbot.database.models.user import User
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Database(apgorm.Database):
+    users = User
+
+    @staticmethod
+    async def create() -> Database:
+        db = Database("mcodingbot/database/migrations")
```

Comment:
Why not class method if it's returning the instance of itself?

Reply:
Because then I would have to write
```py
@classmethod
async def create(cls) -> ???:
    ...
```

Either I need a TypeVar, or I need to specific the return type is Database. And if I'm already using Database explicitly, it felt more consistent to use that too.

If you still think it should be a classmethod I'll change it.

Reply:
I think a classmethod is better

Reply:
I agree with trag1c and lunar here. It makes more sense and is more clear.